<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>420</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>HouseLens Uploader Preference</string>
  </property>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="serverTab">
    <attribute name="title">
     <string>Server</string>
    </attribute>
    <widget class="QTableView" name="serverTableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>390</width>
       <height>220</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FTP Servers:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addServerButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeServerButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="extensionTab">
    <attribute name="title">
     <string>Extension</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add/Remove extensions for Camera Raw files.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableView" name="extTableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>390</width>
       <height>220</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>130</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="addExtensionButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeExtensionButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="dimensionTab">
    <attribute name="title">
     <string>Dimension</string>
    </attribute>
    <widget class="QLineEdit" name="HDHeightTextEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HD Video Dimension</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="HDWidthTextEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="MaxPixelHeightTextEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="DPICompressCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Compress Image If DPI is greater than:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="SizeCompressCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Compress Image If size is greater than:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="MaxPixelWidthTextEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="MaxDPITextEdit">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="dngTab">
    <attribute name="title">
     <string>DNG</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>80</y>
       <width>410</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Preview/Fast Load Data</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>JPEG Preview:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="jpegPreviewComboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>27</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium Size</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Full Size</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="embedFastLoadDataCheckBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>156</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Embed Fast Load Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>381</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>JPEG previews speed display of rendered images. Fast load data speeds loading images when adjusting settings. Both increase DNG file size slightly.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>200</y>
       <width>410</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Compression/Image Size</string>
     </property>
     <widget class="QComboBox" name="lossyCompressionComboBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>28</y>
        <width>230</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Preserve Pixel Count</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Sizes to 600 pixels/side</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Sizes to 800 pixels/side</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Sizes to 1024 pixels/side</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Sizes to 1440 pixels/side</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Sizes to 1680 pixels/side</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Sizes to 2048 pixels/side</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Sizes to 2560 pixels/side</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 1.0MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 1.5MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 2.0MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 3.0MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 4.0MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 6.0MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 8.0MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 10MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 15MP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limit Pixel Count to 20MP</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="useLossyCompressionCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>156</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Lossy compression:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>381</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lossy compression can significantly reduce DNG file size, but may result in some quality loss. Lossy compression also allows optional reduction of image pixel count.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="chooseDNGConverterButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose Adobe DNG Converter</string>
     </property>
    </widget>
    <widget class="QLabel" name="dngConverterPathLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>C:\Program Files\Adobe\Adobe DNG Converter.exe</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
